packages/react/src/patterns/sub-patterns/LinkList/README.md

```diff
@@ -69,10 +79,11 @@ Add the following line in your `.env` file at the root of your project.
 
 ## Props
 
-| Name      | Required | Data Type | Default Value | Description                                          |
-| --------- | -------- | --------- | ------------- | ---------------------------------------------------- |
-| `heading` | YES      | String    | text          | Describes heading of LinkList.                       |
-| `items`   | YES      | Array     | null          | Describes the list of CTA. For more See below `CTA`. |
+| Name      | Required | Data Type | Default Value | Description                                                            |
+| --------- | -------- | --------- | ------------- | ---------------------------------------------------------------------- |
+| `heading` | YES      | String    | text          | Describes heading of LinkList.                                         |
+| `items`   | YES      | Array     | null          | Describes the list of CTA. For more See below `CTA`.                   |
+| `style`   | YES      | String    | null          | Orientation of LinkList. Supports `card`, `horizontal`, and `vertical` |
```

Comment:
@kennylam just doublechecking, doesn't `LinkList` support horizontal and vertical for both card and text type?

Reply:
@jeffchew Confirmed that not all styles can be both orientations. For example, card can only be vertical.